projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -34,7 +34,8 @@
       https://devblogs.microsoft.com/dotnet/embracing-nullable-reference-types/#what-should-library-authors-do
       Note: only setting language version 8.0 for nullable reference types!
```

Comment:
is this comment still necessary? If so, should the C# version be updated to 9.0 in this line?